src/Components/Game/Game.js

```diff
@@ -19,21 +22,15 @@ const Game = ({ get4RdmPokemon}) => {
     })
   }
 
-  const playAudio = () => {
-  const whosThat = document.getElementsByClassName('pokemonSound')[0]
-  if(whosThat){
-    whosThat.play()
-    }
-  }
-
   const checkForWin = (event) => {
     const winner = singlePokemon.name;
     if(winner === event.target.id){
       setWinCounter(winCounter + 1)
-      if(winCounter > bestCounter){
+      if(winCounter >= bestCounter){
```

Comment:
Me and edwin changed this so we don't need nested if statements, I think that is causing merge conflicts